src/infrastructure/gateway/utils.ts

```diff
@@ -0,0 +1,21 @@
+export const convertDateToIsoDateString = (day: string, month: string, year: string): string => {
```

Comment:
should these have an accompanying unit test or are they sufficiently covered by the controller tests?

Reply:
Yes, probably. Will add test for both the utils functions.

Reply:
Tests added.